drivers/timer/stm32_lptim_timer.c

```diff
@@ -550,7 +550,7 @@ static int sys_clock_driver_init(void)
 	return 0;
 }
 
-void sys_clock_idle_exit(void)
+void stm32_clock_control_standby_exit(void)
```

Comment:
There needs another small change here because we relied on the fact that sys_clock_idle_exit was called twice. First to reinit the LPTIM, second to restore the correct elapsed time after the RTC register clock was reenabled.
So we would need something like this:
```
void stm32_clock_control_standby_exit(void)
{
#ifdef CONFIG_STM32_LPTIM_STDBY_TIMER
	if (clock_control_get_status(clk_ctrl,
				     (clock_control_subsys_t) &lptim_clk[0])
				     != CLOCK_CONTROL_STATUS_ON) {
		sys_clock_driver_init();
	}
#endif
}

void sys_clock_idle_exit(void)
{
#ifdef CONFIG_STM32_LPTIM_STDBY_TIMER
	if (timeout_stdby) {
...
```

Reply:
yay ! have fixed it. Please take a look again.